packages/react-components/react-tabster/src/hooks/useModalAttributes.ts

```diff
@@ -18,7 +18,7 @@ export interface UseModalAttributesOptions {
    * Enabling `legacyTrapFocus` prevents users from tabbing out of the focus trap and into
    * the actual browser. Prefer using `trapFocus` instead of this prop.
    */
-  legacyTrapFocus?: boolean;
```

Comment:
you need to deprecate `legacyTrapFocus` since this hook is exported from react-components 🙃